setup.py

```diff
@@ -5,15 +5,15 @@
 requirements = [
     'createrepo_c~=0.13',
     'productmd',
-    'pulpcore-plugin~=0.1rc5',
+    'pulpcore-plugin==0.1.0rc6',
```

Comment:
I think this should be `=` and not `==`.

Reply:
Nevermind, it is `==`.

Reply:
heh, I wrote initially = and it was a wrong syntax ;)